apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.ts

```diff
@@ -0,0 +1,319 @@
+            if (activeCompany && !this.companyUniqueName) {
+                this.companyUniqueName = activeCompany.uniqueName;
+                this.baseCurrencySymbol = activeCompany.baseCurrencySymbol;
+                this.getFormControl('companyUniqueName').patchValue(activeCompany.uniqueName);
+            }
+        });
+    }
+
+    /**
+    * Lifecycle hook for initialization
+    *
+    * @memberof VatLiabilitiesPayments
+    */
+    public ngOnInit(): void {
+        document.querySelector('body').classList.add('gst-sidebar-open');
+        this.getUniversalDatePickerDate();
+        this.activatedRoute.url.pipe(takeUntil(this.destroyed$)).subscribe(params => {
+            this.isPaymentMode = this.router.routerState.snapshot.url.includes('payments');
+            this.displayColumns = this.isPaymentMode ? this.paymentColumns : this.liabilityColumns;
+            this.noDataFound = true;
+            if (this.getFormControl('taxNumber').value) {
+                this.getLiabilitiesPayment();
+            }
+        })
+        this.loadTaxDetails();
+        this.isCompanyMode = this.generalService.currentOrganizationType === OrganizationType.Company;
+
+        if (this.isCompanyMode) {
+            this.currentCompanyBranches$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
+                if (response) {
+                    if (response?.length > 1) {
+                        this.isMultipleBranch = true;
+                        let unarchivedBranches = response.filter(branch => branch.isArchived === false);
+                        this.branchList = unarchivedBranches?.sort(this.generalService.sortBranches);
+                        this.branchList = this.branchList.map(branch => {
+                            return {
+                                label: branch?.alias,
+                                value: branch?.uniqueName
+                            };
+                        });
+                    } else {
+                        this.isMultipleBranch = false;
+                        if (response.uniqueName) {
+                            this.getFormControl('branchUniqueName').patchValue(response.uniqueName);
+                        }
+                    }
+                }
+            });
+        } else {
+            this.getFormControl('branchUniqueName').patchValue(this.generalService.currentBranchUniqueName);
+        }
+    }
+
+    /**
+    * VAT Liabilities/Payments API Call
+    *
+    * @memberof VatLiabilitiesPayments
+    */
+    public getLiabilitiesPayment(): void {
+        this.isLoading = true;
+        let payload = this.generalService.getUserAgentData();
+        payload["Gov-Test-Scenario"] = "MULTIPLE_PAYMENTS_2018_19"
+        this.vatService.getPaymentLiabilityList(payload, this.searchForm.value, this.isPaymentMode ).pipe(takeUntil(this.destroyed$)).subscribe(response => {
```

Comment:
confirm with @ravindersinghdang sir for directly using service not using store now.